tests/unit/test_file_browser_api.py

```diff
@@ -386,7 +389,7 @@ def test_workspace_tree_contents(workspace):
     with pytest.raises(Exception):
         tree.get_path("some_dir").contents()
 
-    tree.get_path("dir.ext").contents() == "dir.ext is not a file"
+    tree.get_path("dir.txt").contents() == "dir.txt is not a file"
```

Comment:
oh nice, this means our empty-dir-with-extension problem is even less likely?

Reply:
yes, seems so